src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_virtual_network.py

```diff
@@ -0,0 +1,209 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from azure.cli.core.commands import LongRunningOperation, _is_poller
+from azure.cli.core.profiles import ResourceType
+from azure.mgmt.resource.resources.models import ResourceGroup
+from ._client_factory import resource_client_factory, network_client_factory
+from msrestazure.tools import is_valid_resource_id, parse_resource_id# pylint: disable=import-error
+from azure.cli.core.commands.client_factory import get_subscription_id
+from azure.cli.core.util import CLIError
+from msrestazure.azure_exceptions import CloudError
+
+logger = get_logger(__name__)
+DEFAULT_VNET_ADDRESS_PREFIX = '10.0.0.0/16'
+DEFAULT_SUBNET_PREFIX = '10.0.0.0/24'
+
+
+def prepareVnet(cmd, server_name, vnet, subnet, resource_group_name, loc, delegation_service_name, vnet_address_pref, subnet_address_pref):
```

Comment:
Please use a pythonic function name.

```suggestion
def prepare_vnet(cmd, server_name, vnet, subnet, resource_group_name, loc, delegation_service_name, vnet_address_pref, subnet_address_pref):
```

Reply:
Updated